packages/core/.storybook/preview.js

```diff
@@ -57,4 +57,4 @@ setCustomElements(customElements);
 // We have this here since storybook does not have a easy way to set the <html> element in demos
 // The token system generates a base 16px set of variables for apps that may not be able to easily set the base font to 125%
 document.body.setAttribute('cds-text', 'body');
-document.querySelector('html').setAttribute('cds-base-font', '16');
+// document.querySelector('html').setAttribute('cds-base-font', '16');
```

Comment:
The tokens support base 20 and base 16 font size. This line enables base 16 in storybook to test with. I forgot to comment it back in from my prior PR after testing.

Reply:
as cory mentioned, i'm commenting out a test that cory had left uncommented in the storybook app. for future reference, it is a good way to test whether your sizings on components is flexible. it will highlight how brittle the CSS code is.